### ISSUE TITLE
Feature Suggestion Add "/" to the keyboard top hotbar

### ISSUE THREAD

**Issue by gt2416** [ish-app/ish#1535]
As the title says, adding "/" will be useful as we use that a lot to move through the filesystem.

**Comment by leo848** [comment]
No, the / symbol is easily accessible via a hardware keyboard. A good option would be to add a button that includes symbols you can customize.

**Comment by Smegheid** [comment]
I’m in agreement with @gt2416 - when used without an external keyboard, having shortcuts to other symbol keys in the hotbar would be _very_ useful.

I’d also go further and request a bunch of symbol shortcuts, such as the vertical pipe symbol, < and > for redirection, backslash for escaping characters, { and } for expansion, etc. These are a pain to get to with the default iOS keyboard.

The hotbar is otherwise unused space - might as well fill it full of as many shortcuts as possible.
